drf_spectacular/views.py

```diff
@@ -261,3 +262,14 @@ def _get_schema_url(self, request):
             lang=request.GET.get('lang'),
             version=request.GET.get('version')
         )
+
+
+class SwaggerOauthRedirectView(RedirectView):
+    """
+    A view that serves the SwaggerUI oauth2-redirect.html file so that SwaggerUI can authenticate itself using Oauth2
+
+    This view should be served as ``./oauth2-redirect.html`` relative to the SwaggerUI itself.
+    If that is not possible, this views absolute url can also be set via the ``SPECTACULAR_SETTINGS.SWAGGER_UI_SETTINGS.oauth2RedirectUrl`` django settings.
```

Comment:
this line is probably too long and flake8 will complain once the test runs

Reply:
fixed in 10dbd5dc631bb590fcaaf01ee7ae8ddafbdc35f6